### PR TITLE
[issue-126] Create Organization Page - Page Forward

### PR DESCRIPTION
**Pull request by ameepandya**
To test,
1) Create an org for admin user
2) Check after create it redirects to view org details page.

Note : There are few issues with this
1) after clicking create button it takes some time to create an org. I think it has some backend issue because  at the same time eclipse gets hang for a while.
2) View org detail page is showing some data like city & zip which are not available in getOrganization() response that's why it shows undefined.

### CODE REVIEW

File: src/app/organization/common/organization.service.ts [hunk 1]
```diff
@@ -19,8 +19,7 @@ export class OrganizationService {
   }
 
   getOrganization(id: number): Observable<Organization> {
-    const index = id - 1;
-    const url = organizationUrl + '/' + index;
+    const url = organizationUrl + '/' + id;
```
Comment (russon77) [inline src/app/organization/common/organization.service.ts]: could be string interpolation with backticks, i.e. `` `${organizationUrl}/${id}` ``

File: src/app/organization/create/organization-create.component.ts [hunk 2]
```diff
@@ -28,10 +29,13 @@ export class OrganizationCreateComponent implements OnInit {
...
-  constructor(public fb: FormBuilder,
-              private organizationService: OrganizationService,
-              private fc: FormConstantsService,
-              private el: ElementRef) { }
+  constructor(
+    public fb: FormBuilder,
+    private router: Router,
+    private organizationService: OrganizationService,
+    private fc: FormConstantsService,
+    private el: ElementRef
```
Comment (russon77) [inline src/app/organization/create/organization-create.component.ts]: is `el: ElementRef` used anywhere?

File: src/app/organization/create/organization-create.component.ts [hunk 3]
```diff
@@ -10,7 +11,7 @@ import { FormConstantsService } from '../../_services/form-constants.service';
 })
 
 export class OrganizationCreateComponent implements OnInit {
-  public categories: String[];
+  public categories: any[];
```
Comment (russon77) [inline src/app/organization/create/organization-create.component.ts]: categories should have a type / model definition

File: src/app/_services/form-constants.service.ts [hunk 4]
```diff
@@ -257,11 +257,11 @@ export class FormConstantsService {
     { code: 'ZMB', name: 'Zambia' },
     { code: 'ZWE', name: 'Zimbabwe' }
   ];
-  private categories = ['Non-Profit'];
+  private categories = [{name: 'Non-Profit', value: 'N'}];
 
   constructor() { }
 
-  getCategories(): string[] {
+  getCategories(): any[] {
```
Comment (russon77) [inline src/app/_services/form-constants.service.ts]: category type/model would be used here too

Comment (ameepandya) [inline src/app/_services/form-constants.service.ts]: private categories = [{name: 'Non-Profit', value: 'N'}];

Categories is actually constant array with key value pairs in it so I don't know how to define type of categories.

Comment (russon77) [inline src/app/_services/form-constants.service.ts]: an easy definition could be 
```
getCategories(): {[key: string]: string} {
```

[sic]

or you could use a `Map<string, string>` type possibly, although i haven't had much experience with that.

**Pull request merged**